java/hibernate/README.md

```diff
@@ -64,5 +64,83 @@ We have generated schema definitions that are currently applied in the database
 ./flway migrate
 ```
 
+## Delete REST API request 
```

Comment:
This could be put above 'Schema Migration' section. And you can mention that in addition to GET/POST, users can send DELETE request to delete records from corresponding tables.